### PR TITLE
homework js3 week1

### PR DESCRIPTION
**Pull request by Nadoshka**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -29,19 +30,79 @@
...
+    createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = createAndAppend('select', header, {
+      id: 'list',
+    });
+    const div = createAndAppend('div', root, { class: 'list-div' });
+    const list = createAndAppend('ul', div, { class: 'list1' });
+    const listContributor = createAndAppend('ul', div, { class: 'list2' });
+    const newList = createAndAppend('li', list, { text: ' ' });
```
Comment (paulbremer) [inline homework/index.js]: Better to use names you and other developers can remember later in time, which one do you think is better; `newList` or `descriptionList`?

File: homework/index.js [hunk 2]
```diff
@@ -29,19 +30,79 @@
...
+    const newList1 = createAndAppend('li', list, { text: ' ' });
+    const newList2 = createAndAppend('li', list, { text: ' ' });
+    const newList3 = createAndAppend('li', list, { text: ' ' });
 
+    for (let i = 0; i < repos.length; i++) {
+      createAndAppend('option', select, { text: repos[i].name, value: i });
+    }
+
+    function leftHand() {
+      newList.innerHTML = `Repository: <a target=_blank href= ${repos[select.value].html_url}>${
```
Comment (paulbremer) [inline homework/index.js]: Modern browsers will add the quotes for you `target=_blank`, but some old browsers might not, so do not forget to add quotes! `target="_blank"` around all the different tags.

File: homework/index.js [hunk 3]
```diff
@@ -29,19 +30,79 @@
...
+      newList1.innerText = `Description: ${repos[select.value].description}`;
+      newList2.innerText = `Forks: ${repos[select.value].forks}`;
+      newList3.innerText = `Updated: ${repos[select.value].updated_at}`;
+      const links = repos[select.value].contributors_url;
+      fetchJSON(links, (err, data) => {
+        if (err) {
+          createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+        } else {
+          let linksContributor = JSON.stringify(data, null, 2);
+          linksContributor = JSON.parse(linksContributor);
```
Comment (paulbremer) [inline homework/index.js]: Why are you stringify'ing and on the next line parsing it again, looks to me there is no need and you can just ust the `data` 💁‍♂️

File: homework/index.js [hunk 4]
```diff
@@ -29,19 +30,79 @@
...
+          listContributor.innerText = ' ';
+          for (let i = 0; i < linksContributor.length; i++) {
+            listContributor.innerHTML += `<li><a target =_blank href= ${
```
Comment (paulbremer) [inline homework/index.js]: Again, don't forget the quotes ""!

File: homework/index.js (hunk 1 above)
Comment (Nadoshka) [reply homework/index.js]: Thanks! you are right! descriptionList will be readable and understandable 👍 :)

File: homework/index.js (hunk 2 above)
Comment (Nadoshka) [reply homework/index.js]: Thanks! I will never forget it :)

File: homework/index.js (hunk 4 above)
Comment (Nadoshka) [reply homework/index.js]: 👍

File: homework/index.js (hunk 3 above)
Comment (Nadoshka) [reply homework/index.js]: you are right no need to do that! I paied my attention to that in the homework week 2 so, I delet it. :)

File: homework/index.js (hunk 4 above)
Comment (Nadoshka) [reply homework/index.js]: Thanks for your feedback! I will pay my attention to it in the next time. :) 👍

Comment (wouterkleijn) [reply homework/index.js]: Thanks for the replies to @paulbremer's comments  - very professional! @paulbremer Do you want to see improvements on the code still or can we close this PR?

File: homework/index.js (hunk 1 above)
Comment (paulbremer) [reply homework/index.js]: Good!

File: homework/index.js (hunk 4 above)
Comment (paulbremer) [reply homework/index.js]: All good! @Nadoshka did the improvements in the week2 homework 🎉This PR can be closed.

**Pull request closed without merging**